go/cli/check-tag/main.go

```diff
@@ -0,0 +1,38 @@
+/// Utility for checking that the current git tag matches the internal MCAP library version string.
```

Comment:
I generally hate bash scripts, but it kinda feels like this whole program could be a few lines of bash. Is there some technical reason to make a separate go binary for this rather than just calling `mcap version --library` to get the library version and checking it against the git tag?